frontend/app-development/layout/SettingsModalButton/SettingsModal/components/Tabs/SetupTab/SetupTabContent/SetupTabContent.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { ReactNode, useState } from 'react';
+import classes from './SetupTabContent.module.css';
+import { ApplicationMetadata } from 'app-shared/types/ApplicationMetadata';
+import { useTranslation } from 'react-i18next';
+import { useAppMetadataMutation } from 'app-development/hooks/mutations';
+import { ErrorMessage, Textfield } from '@digdir/design-system-react';
+import { SwitchRow } from './SwitchRow';
+import { Divider } from 'app-shared/primitives';
+import { getIsDatesValid } from 'app-development/layout/SettingsModalButton/SettingsModal/utils/tabUtils/setupTabUtils';
+
+export type SetupTabContentProps = {
+  appMetadata: ApplicationMetadata;
+  org: string;
+  app: string;
+};
+
+/**
+ * @component
+ *    The content of the Setup Tab
+ *
+ * @property {ApplicationMetadata}[appMetadata] - The application metadata
+ * @property {string}[org] - The org
+ * @property {string}[app] - The app
+ *
+ * @returns {ReactNode} - The rendered component
+ */
+export const SetupTabContent = ({ appMetadata, org, app }: SetupTabContentProps): ReactNode => {
+  const { t } = useTranslation();
+
+  const [startDate, setStartDate] = useState<string>(appMetadata?.validFrom ?? '');
+  const [endDate, setEndDate] = useState<string>(appMetadata?.validTo ?? '');
```

Comment:
Burde ikke disse være string | undefined? 😊 Disse verdiene brukes videre i utils-metoden `getIsDatesValid`, og det første den sjekker er om `(from === undefined || to === undefined) return true;`. Siden de har default tom string, så treffer ikke from og to sjekken på undefined. 

Håper jeg klarte å formulere med forståelig. 🙈 

```suggestion
  const [startDate, setStartDate] = useState<string | undefined>(appMetadata?.validFrom);
  const [endDate, setEndDate] = useState<string | undefined>(appMetadata?.validTo);
```

Reply:
Ideelt sett ja, men da får du feil om at verdien endres fra en ukontrollert input til en kontrollert en. Dette skjer når det er undefined og null. Dersom man har en tom string, så forsvinner denne feilen. 
<img width="531" alt="image" src="https://github.com/Altinn/altinn-studio/assets/133344438/224bcb45-d423-4875-bfbf-b5e4f51cd2e3">

Reply:
Jeg så på dette sammen med William. Grunnen til at vi løste det på denne måten er at React tolker `input` som ukontrollert hvis `value` settes til `undefined`. Hvis man velger å fjerne datoen i nettleserverktøyet, blir den også satt tilbake til en tom streng, så det er nok heller valideringsfunksjonen vi burde tilpasse her.

Reply:
Ja, det gir mening når jeg ser kommentaren til @WilliamThorenfeldt. 😊